gerrit-entrypoint.sh

```diff
@@ -9,6 +9,14 @@ set_secure_config() {
   su-exec ${GERRIT_USER} git config -f "${GERRIT_SITE}/etc/secure.config" "$@"
 }
 
+wait_for_database() {
+  echo "Waiting for database connection $1:$2 ..."
+  until nc -z $1 $2; do
+    sleep 1
+  done
+  sleep 2
```

Comment:
The above is not my idea, I saw it [here](https://github.com/mattermost/mattermost-docker/blob/4.1.0/app/docker-entry.sh). They use an additional `sleep 1` for postgres to avoid some error (as mentioned in the comment). I was lazy and did not test the behavior of MySQL. So I just added another second for safety. Does not hurt. But we can use `sleep 1`. No prob.